pkg/apis/serving/v1beta1/component.go

```diff
@@ -30,15 +30,19 @@ import (
 
 // Known error messages
 const (
-	MinReplicasShouldBeLessThanMaxError = "'MinReplicas' cannot be greater than MaxReplicas"
-	MinReplicasLowerBoundExceededError  = "'MinReplicas' cannot be less than 0"
-	MaxReplicasLowerBoundExceededError  = "'MaxReplicas' cannot be less than 0"
-	ParallelismLowerBoundExceededError  = "parallelism cannot be less than 0"
-	UnsupportedStorageURIFormatError    = "storageUri, must be one of: [%s] or match https://{}.blob.core.windows.net/{}/{} or be an absolute or relative local path. StorageUri [%s] is not supported"
-	UnsupportedStorageSpecFormatError   = "storage.spec.type, must be one of: [%s]. storage.spec.type [%s] is not supported"
-	InvalidLoggerType                   = "invalid logger type"
-	InvalidISVCNameFormatError          = "the InferenceService \"%s\" is invalid: a InferenceService name must consist of lower case alphanumeric characters or '-', and must start with alphabetical character. (e.g. \"my-name\" or \"abc-123\", regex used for validation is '%s')"
-	InvalidProtocol                     = "invalid protocol %s. Must be one of [%s]"
+	MinReplicasShouldBeLessThanMaxError   = "'MinReplicas' cannot be greater than MaxReplicas"
+	MinReplicasLowerBoundExceededError    = "'MinReplicas' cannot be less than 0"
+	MaxReplicasLowerBoundExceededError    = "'MaxReplicas' cannot be less than 0"
+	ParallelismLowerBoundExceededError    = "parallelism cannot be less than 0"
+	UnsupportedStorageURIFormatError      = "storageUri, must be one of: [%s] or match https://{}.blob.core.windows.net/{}/{} or be an absolute or relative local path. StorageUri [%s] is not supported"
+	UnsupportedStorageSpecFormatError     = "storage.spec.type, must be one of: [%s]. storage.spec.type [%s] is not supported"
+	InvalidLoggerType                     = "invalid logger type"
+	InvalidISVCNameFormatError            = "the InferenceService \"%s\" is invalid: a InferenceService name must consist of lower case alphanumeric characters or '-', and must start with alphabetical character. (e.g. \"my-name\" or \"abc-123\", regex used for validation is '%s')"
+	InvalidProtocol                       = "invalid protocol %s. Must be one of [%s]"
+	InvalidWorkerSpecSizeValueError       = "the InferenceService \"%s\" is invalid: WorkerSpec.Size cannot be less than 2(%d)"
```

Comment:
Can we also mention that this is for multi-node only?

Reply:
I believe we should avoid adding that clarification, as the error message is intended to be generic. It does not specify the context, such as serverless, RawDeployment, or specific ServingRuntime configurations. Multi-node is just one of many features, and specifying it in the error message could lead to confusion or misinterpretation in different contexts.

Reply:
Is this less than 1? The check below is `WorkerSpecSize < 1`

Reply:
good catch! updated